substrate/frame/contracts/src/benchmarking/mod.rs

```diff
@@ -332,7 +332,7 @@ mod benchmarks {
 	#[benchmark(pov_mode = Measured)]
 	fn migration_noop() {
 		let version = LATEST_MIGRATION_VERSION;
-		assert_eq!(StorageVersion::get::<Pallet<T>>(), version);
+		StorageVersion::new(version).put::<Pallet<T>>();
```

Comment:
seems like a totally different logic now, intended?

Reply:
Yea these random changes are because it is using the runtime genesis and not the chain-spec genesis.  
The benchmark should still do the same.